apps/dev-recruiters/src/components/modules/FilterPage/RolesFilterComponent/RolesFilterList/CollapsibleContainerFilter/index.tsx

```diff
@@ -23,11 +23,18 @@ const CollapsibleContainerFilter: React.FC<CollapsibleContainerProps> = ({
 }) => {
   const [roleActive, setRoleActive] = useState(null);
 
-  function handleRoleClick(roleName, roleLabel) {
+  useEffect(() => {
+    const roleName = localStorage.getItem('roleName');
+    setRoleActive(roleName);
+  }, []);
+
+  function handleRoleClick(roleName, roleLabel, roleCategory) {

```

Comment:
Is `roleLabel` the `roleType` in the backend https://github.com/dev-launchers/strapiv4/blob/main/src/api/opportunity/content-types/opportunity/schema.json#L70?
If so, I would recommend calling them both roleType to be consistent.

Reply:
No its different. roleLabel contains all the opportunities list under that roleType